### ISSUE TITLE
Add syntax highlighting

### ISSUE THREAD

**Issue by HarshKapadia2** [HarshKapadia2/git-graph#22]
- In the `RawDataDisplay` component, add syntax highlighting to the Object Content field.
- One way to implement it could be to extract the file extension form the Object Name field and use that to identify the language.
	- Eg: `script.js` ➡️ `js` ➡️ `javascript`
		- Maybe `js` itself will work and the last transformation of `js` to `javascript` might not be required, but then would `mjs` work?
- A dynamic syntax highlighter would be required, not a static one, because nothing is server rendered and files can be of any language.
- Language support will have to be decided.
	- Maybe HTML, CSS, JS, Shell, Bash, YAML and Python for now.

**Comment by tusharnankani** [comment]
> Language support will have to be decided.

It will be based on the package you will use. Why fix specific languages? 
Why not give all the support we have?

**Comment by HarshKapadia2** [comment]
> It will be based on the package you will use.

True, so if all languages will be supported dynamically, I will obviously go for that.

> > Language support will have to be decided.
>
> It will be based on the package you will use. Why fix specific languages?

The reason I mentioned language support is that some highlighters make one include a JS bundle (through links in `index.html`) for every language one wants to support, so I was just ensuring that I have a list ready if that is the case. I should've written that down. My bad. Thank you for clarifying, @tusharnankani! (This case would suck for performance, but can't help it if I'm not able to find anything.)
